chia/wallet/puzzles/condition_codes.clvm.hex

```diff
@@ -0,0 +1 @@
+can't compile ("defconstant" "AGG_SIG_UNSAFE" 49), unknown operator
```

Comment:
The compiler is only compiling this because it has a `.clvm` extension but in reality it's a `.clib` or a file that's meant to be included rather than compiled independently.  If we change the file extensions these will go away but I suppose it wouldn't hurt to fix the test as well.